app/src/main/java/org/mozilla/focus/fragment/onboarding/OnboardingFirstFragment.kt

```diff
@@ -21,6 +23,12 @@ import org.mozilla.focus.ui.theme.FocusTheme
 class OnboardingFirstFragment : Fragment() {
     private lateinit var onboardingInteractor: OnboardingInteractor
 
+    @SuppressLint("SourceLockedOrientationActivity")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        requireActivity().requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
```

Comment:
Can we move this logic to the controller? In maybe "handleOnboardingStarted"?
Then the controller would be the only one handling locking and unlocking orientations.

Reply:
Done